.gitlab-ci.yml

```diff
@@ -8,15 +8,14 @@ stages:
 before_script:
   - nix-shell --run "yarn"
 
-# Build & Lint
-flow lint:
+verify:
   stage: build
   script:
     - nix-shell --run "yarn flow"
+    - nix-shell --run "yarn typescript"
```

Comment:
add this job to github actions as well:
https://github.com/trezor/connect/blob/develop/.github/workflows/tests.yml#L41

Reply:
added in https://github.com/trezor/connect/pull/785/commits/f513e33a7094ef1d6a400af511e732e3857130ee